_posts/0000-01-11-hiltonle.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+No Change 
```

Comment:
Maybe add a interesting saying for your image.

Reply:
the image? it is CN Tower